Source/JavaScriptCore/wasm/WasmCallee.h

```diff
@@ -53,7 +53,7 @@ class LLIntOffsetsExtractor;
 namespace Wasm {
 
 class Callee : public NativeCallee {
-    WTF_MAKE_TZONE_ALLOCATED(Callee);
+    WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER(Callee);
```

Comment:
Why is this needed?  Let's ping Michael on this.

Reply:
The new JIT callee will allocate metadata directly at the end of Callee, which I am assuming doesn't fit the goal of TZone

Reply:
Please add a comment in the base class JITCallee to indicate the subclass that is variable sized.  That way, someone won't innocently put this back in the TZone.